conf/evolutions/bakery_db/1.sql

```diff
@@ -0,0 +1,18 @@
+-- bakery_db schema
+
+-- !Ups
+
+CREATE TABLE IF NOT EXISTS Bakery
+(
+    id uuid NOT NULL DEFAULT gen_random_uuid(),
+    name character varying COLLATE pg_catalog."default" NOT NULL,
+    quantity integer,
+    price double precision,
+    "created at" timestamp without time zone,
+    "updated at" timestamp without time zone,
```

Comment:
created_at and updated_at are preferable, easier to query

Reply:
That makes sense, big mistake from my end, will correct!